napi-inl.h

```diff
@@ -572,6 +572,14 @@ void Env::DefaultFiniWithHint(Env, DataType* data, HintType*) {
 }
 #endif  // NAPI_VERSION > 5
 
+#if NAPI_VERSION > 8
+inline std::string_view Env::GetModuleFileName() const {
+  const char* result;
+  napi_status status = node_api_get_module_file_name(_env, &result);
```

Comment:
Node.js v16.x is still compiled with [C++14](https://github.com/nodejs/node/blob/v16.x/common.gypi#L405), however `std::string_view` is available since c++17. I think we should avoid using c++17 features here.

Reply:
Mhmm mhmm that makes sense ... So then would we prefer the alternative of node-addon-api returning a new `std::string` by copying the `const char* result`?

Reply:
Would it be good to return the `const char* result`? If needed, copying the result with `std::string` is fairly easy.